docs/swagger_v3/oracles.spec.yaml

```diff
@@ -1,17 +1,73 @@
 schemas:
+  OracleTx:
```

Comment:
Same here, what's an OracleTx? If it's an extends tx, shouldn't it be OracleExtendsTx ?

Reply:
I'm using it only for the shape of the object, so it's not written everytime it's being used